CloudFabric.EventSourcing.EventStore/Persistence/ItemWrapper.cs

```diff
@@ -0,0 +1,15 @@
+using System.Text.Json.Serialization;
+
+namespace CloudFabric.EventSourcing.EventStore.Persistence;
+
+public class ItemWrapper
+{
+    [JsonPropertyName("id")]
+    public string? Id { get; set; }
+
+    [JsonPropertyName("partitionKey")]
```

Comment:
this does not match with item_data naming convention, please use the one which is used in events table